src/python/WMComponent/RucioInjector/RucioInjectorPoller.py

```diff
@@ -78,6 +79,9 @@ def __init__(self, config):
                            authUrl=config.RucioInjector.rucioAuthUrl,
                            configDict={'logger': self.logger})
 
+        self.deep = getattr(config.RucioInjector, "deepFlag", False)
```

Comment:
What about changing the name of this flag to something more descriptive. like `useDsetReplicaDeep` ?

Reply:
Maybe even `useDsetReplicaDeepFlag`